book-a-reading-room-visit.web/Helper/Constants.cs

```diff
@@ -24,5 +24,7 @@ public class Constants
         public const string CookieName = "cookies_policy";
         public const string Localhost = "localhost";
         public const string TNADomain = "nationalarchives.gov.uk";
+
+        public const string Invalid_Document_Reference = "The document reference you have entered has an invalid format.  References must start with 1-4 letters, followed by a space, then 1-4 digits";
```

Comment:
@Brian-O-TNA as per this comment in the Jira ticket a Series can be 5 digits in length. So the validation message should be "... References must start with 1-4 letters, followed by a space, then 1-5 digits";
https://national-archives.atlassian.net/browse/ACCESS-222?focusedCommentId=33880

Reply:
I checked ILDB. class_id 15020 has letter code TEMP. There are 4 of them TEMP 15020, 15019, 11010 and 11005.

The next highest is, as you say Matt, DF 5012.

So the validation message is correct as it stands. No need to change.